container/Containerfile

```diff
@@ -0,0 +1,233 @@
+    #       gcc, lua-devel, luarocks
+    #    They are listed as BuildRequires in ceph.spec.in, so I assume they are unnecessary
+    #    @mattbenjamin please advise about whether these are necessary for runtime
+    PACKAGES="$PACKAGES \
+        ceph-common${CEPH_PACKAGE_VERSION} \
+        ceph-exporter${CEPH_PACKAGE_VERSION} \
+        ceph-grafana-dashboards${CEPH_PACKAGE_VERSION} \
+        ceph-immutable-object-cache${CEPH_PACKAGE_VERSION} \
+        ceph-mds${CEPH_PACKAGE_VERSION} \
+        ceph-mgr-cephadm${CEPH_PACKAGE_VERSION} \
+        ceph-mgr-dashboard${CEPH_PACKAGE_VERSION} \
+        ceph-mgr-diskprediction-local${CEPH_PACKAGE_VERSION} \
+        ceph-mgr-k8sevents${CEPH_PACKAGE_VERSION} \
+        ceph-mgr-rook${CEPH_PACKAGE_VERSION} \
+        ceph-mgr${CEPH_PACKAGE_VERSION} \
+        ceph-mon${CEPH_PACKAGE_VERSION} \
+        ceph-osd${CEPH_PACKAGE_VERSION} \
+        ceph-radosgw${CEPH_PACKAGE_VERSION} \
+        ceph-volume${CEPH_PACKAGE_VERSION} \
+        cephfs-mirror${CEPH_PACKAGE_VERSION} \
+        cephfs-top${CEPH_PACKAGE_VERSION} \
+        kmod \
+        libradosstriper1${CEPH_PACKAGE_VERSION} \
+        rbd-mirror${CEPH_PACKAGE_VERSION} \
+        $(cat crimson-packages.txt) \
+        " && \
+    # Ceph "Recommends"
+    PACKAGES="$PACKAGES \
+        nvme-cli \
+        python3-saml \
+        smartmontools \
+        " && \
+    # NFS-Ganesha
+    PACKAGES="$PACKAGES \
+        dbus-daemon \
+        nfs-ganesha-ceph${GANESHA_PACKAGE_VERSION} \
+        nfs-ganesha-rados-grace${GANESHA_PACKAGE_VERSION} \
+        nfs-ganesha-rados-urls${GANESHA_PACKAGE_VERSION} \
+        nfs-ganesha-rgw${GANESHA_PACKAGE_VERSION} \
+        nfs-ganesha${GANESHA_PACKAGE_VERSION} \
+        rpcbind \
+        sssd-client \
+        " && \
+    # ISCSI
+    PACKAGES="$PACKAGES \
+        ceph-iscsi \
+        tcmu-runner \
+        python3-rtslib \
+        " && \
+    # Ceph-CSI
+    # TODO: coordinate with @Madhu-1 to have Ceph-CSI install these itself if unused by ceph
+    #       @adk3798 does cephadm use these?
+    PACKAGES="$PACKAGES \
+        attr \
+        ceph-fuse${CEPH_PACKAGE_VERSION} \
+        rbd-nbd${CEPH_PACKAGE_VERSION} \
+        " && \
+    # Rook (only if packages must be in ceph container image)
+    PACKAGES="$PACKAGES \
+        systemd-udev \
+        " && \
+    # Util packages (should be kept to only utils that are truly very useful)
+    # 'sgdisk' (from gdisk) is used in docs and scripts for clearing disks (could be a risk? @travisn @guits @ktdreyer ?)
+    # 'ps' (from procps-ng) and 'hostname' are very valuable for debugging and CI
+    # TODO: Per discussion, sg3_utils should be moved to ceph.spec.in with libstoragemgmt; @ktdreyer
+    #       ref: https://github.com/ceph/ceph-container/pull/2013#issuecomment-1248606472
+    PACKAGES="$PACKAGES \
+        gdisk \
+        hostname \
+        procps-ng \
+        sg3_utils \
```

Comment:
@ktdreyer FYI, it seems like there will be a follow-up after this to mark sg3_utils as an RPM "Requires".